tasks/packages.yml

```diff
@@ -21,21 +21,6 @@
   tags:
     - r-packages-copy-r-scripts
 
-- name: packages | install
-  command: >
-    R-install-package
-      {{ item.name }}
-      {{ item.type | default(r_packages_type) }}
-      {{ item.lib | default(r_packages_lib) }}
-      {% if item.repos is defined %}{{ item.repos }}{% endif %}
-  environment: "{{ r_environment }}"
-  register: _r_install_package
-  changed_when: "_r_install_package.stdout_lines[-1] is defined and _r_install_package.stdout_lines[-1] == 'changed'"
-  with_items: "{{ r_packages_from_github | ternary(r_preset_package_remotes + r_packages, r_packages) }}"
-  when: item.state is undefined or item.state == 'present'
-  tags:
-    - r-packages-install
-
```

Comment:
Why is this task moved?

Reply:
Because I needed to update a package before installing a new one.

Reply:
this will failed on installation without package